Updates for withdrawing functionality (#19)
gnosis/dex-docs: docs/tutorial-cmm.md

Transaction awaiting execution in the interface https://rinkeby.gnosis-safe.io/app/#/safes/$MASTER_SAFE/transactions
```

> Now, you should just follow the link, and sign and execute the transaction with the lower transaction-ID first. It should look like this:

Reviewer: ```suggestion
Now, you should just follow the link, and sign and execute the transaction with the lower transaction ID first. It should look like this:
```

Withdrawal requests can be made with the following command:

```ssh

> npx truffle exec scripts/request_withdraw.js --masterSafe=$MASTER_SAFE --brackets=[comma-separated-brackets]  --tokenIds=[indices] --network=$NETWORK_NAME

Reviewer: ```suggestion
npx truffle exec scripts/request_withdraw.js --masterSafe=$MASTER_SAFE --brackets=[comma-separated brackets]  --tokenIds=[comma-separated indices] --network=$NETWORK_NAME
```

```

> The `tokenIndices` of the tokens, which you want to withdraw from the brackets, need to be specified via the flag `--tokenIds=[indices]`. Do **not** write the square brackets into the command. Under normal usage, these are the exact same indices that were provided when setting up the CMM.

Reviewer: ```suggestion
The indices of the tokens that you want to withdraw from the brackets need to be specified via the flag `--tokenIds=[comma-separated indices]`. Do **not** write the square brackets into the command. Under normal usage, these are the exact same indices that were provided when setting up the CMM.
```

> The flag `--brackets=[comma separated brackets]` must contain the brackets that the CMM deployed. Please provide these addresses separated by commas, without any spaces, and without any square brackets.

Reviewer: ```suggestion
The flag `--brackets=[comma-separated brackets]` must contain the brackets that the CMM deployed. Please provide these addresses separated by commas, without any spaces, and without any square brackets.
```

The actual withdrawal transaction transferring the tokens back into the MASTER_SAFE from Gnosis Protocol can be initiated with the following command.

```ssh

> npx truffle exec scripts/claim_withdraw.js --masterSafe=$MASTER_SAFE --brackets=[comma-separated-brackets]  --tokenIds=[indices] --network=$NETWORK_NAME

Reviewer: ```suggestion
npx truffle exec scripts/claim_withdraw.js --masterSafe=$MASTER_SAFE --brackets=[comma-separated brackets]  --tokenIds=[comma-separated indices] --network=$NETWORK_NAME
```
